File: java/11653 소인수분해.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        int n = scn.nextInt();
        int div = 2;
        Vector<Integer> v = new Vector<Integer>();

        while (n != 1)
        {
            if (n % div == 0)
            {
                v.add(div);
                n /= div;
            }
            else
                div++;
        }

        for (int i = 0; i < v.size(); i++)
            System.out.println(v.get(i));

        scn.close();
    }
}

File: 파이썬/11653 소인수분해.py
n = int(input())
div = 2

while n != 1:
    if n % div == 0:
        print(div)
        n //= div  # "n /= div"라고 써도 결과는 정상적으로 출력되지만 n이 실수형으로 바뀐다
    else:
        div += 1
